specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/extensions/readme.go.md

```diff
@@ -0,0 +1,28 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go) && !$(track2)
+go:
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: kubernetesconfiguration
+  clear-output-folder: true
+```
+
+``` yaml $(go) && $(track2)
+license-header: MICROSOFT_MIT_NO_VERSION
+module-name: sdk/resourcemanager/kubernetesconfiguration/armkubernetesconfiguration
```

Comment:
Looks like all of the SDKs have the same value for `module-name` so they're stomping on each other.

Since the intention is to have separate SDKs for each service, the names will need to be unique.

e.g. `sdk/resourcemanager/kubernetesconfiguration/arm<something unique here>` assuming they're all under the `kubernetesconfiguration` path.